app/models/changeset/pull_request.rb

```diff
@@ -31,6 +33,17 @@ def self.find(repo, number)
     nil
   end
 
+  def self.changeset_from_webhook(project, params = {})
+    find(project.github_repo, params[:number])
+  end
+
+  def self.valid_webhook?(params)
+    data = params[:pull_request] || {}
+    return false unless data[:state] == 'open'
+
+    data[:body] =~ WEBHOOK_FILTER || data.fetch(:base, {})[:ref] == 'production'
```

Comment:
the description said it would ignore production ... but this does the opposite ?
 - missing test ?
 - will cause problems when using other branches like staging to pick commits ?